src/components/Numbers/index.jsx

```diff
@@ -90,29 +59,36 @@ const Numbers = () => {
     );
   }
 
-  if (error) {
+  if (dataError) {
```

Comment:
Error condition should be before Loading condition

Reply:
I tried both ways. and they are working without issues. But I will check the error first.